Tests/js/views/services/assets/ez-discoverybarviewservice-tests.js

```diff
@@ -131,11 +131,24 @@ YUI.add('ez-discoverybarviewservice-tests', function (Y) {
         },
         
         "Should fire contentDiscover with a starting location on browseAction event": function () {
-            var contentDiscoverFired = false;
+            var contentDiscoverFired = false,
+                otherLocation = new Y.Base();
 
+            otherLocation.set('id', 'AnyOtherId');
             this.service.on('contentDiscover', Y.bind(function (e) {
                 contentDiscoverFired = true;
 
+                Assert.isFunction(e.config.isSelectable, "config should have a function named isSelectable");
+
+                Y.Assert.isTrue(
+                    e.config.isSelectable({location: otherLocation}),
```

Comment:
please add 2 new test cases that would call the `isSelectable` function and make sure the starting Location can not be selected and any other Location is selectable.